pulp.spec

```diff
@@ -325,6 +325,7 @@ touch %{buildroot}/%{_sysconfdir}/pki/%{name}/consumer/server/rsa_pub.key
 cp -R agent/etc/pulp/agent/agent.conf %{buildroot}/%{_sysconfdir}/%{name}/agent/
 cp -R client_consumer/etc/pulp/consumer/consumer.conf %{buildroot}/%{_sysconfdir}/%{name}/consumer/
 %if 0%{?rhel} >= 6 || 0%{?fedora} >= 19
+mkdir -p %{buildroot}/%{_sysconfdir}/bash_completion.d
```

Comment:
This is a "when in rome" change, so this bit tries to look a lot like other places in the spec file that do the same thing (it is, in fact, a copy of another line from another section) due to a well-earned fear of making changes to our pulp 2 specs. :)